### PR TITLE
Change grammar of sentences

### PR DESCRIPTION
**Pull request by Valdnet**
Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

File: src/components/ConversationSettings/ConversationPermissionsSettings.vue [hunk 1]
```diff
@@ -38,7 +38,7 @@
 			</CheckboxRadioSwitch>
 			<span v-show="loading && radioValue === 'all'" class="icon-loading-small" />
 		</div>
-		<p>{{ t('spreed', 'Participants have permissions to start a call, join a call, enable audio, video and screenshare.') }}</p>
+		<p>{{ t('spreed', 'Participants have permissions to start a call, join a call, enable audio and video, and screenshare.') }}</p>
```
Comment (nickvergessen) [inline src/components/ConversationSettings/ConversationPermissionsSettings.vue:41]: It's a separate permission, so i would use "audio and video" and then comma separate screenshare

Comment (Valdnet) [reply src/components/ConversationSettings/ConversationPermissionsSettings.vue]: Dane 😉.

**Comment by rakekniven** [comment]
It would be good if a english native speaker would read and review this sentence.

Any recommendation?

**Review by marcoambrosini — APPROVED** [review]
All good other than we invented the verb to screenshare. We should probably change it to share the screen

**Comment by Valdnet** [comment]
@marcoambrosini Changes made 😉.

**Comment by rakekniven** [comment]
> @marcoambrosini Changes made 😉.

There are more occurences of "screenshare" in this app.
See https://github.com/nextcloud/spreed/search?q=screenshare

If you change this, we should be like "all or nothing". 😄

**Review by rakekniven — CHANGES_REQUESTED** [review]
See my last comment

**Comment by Valdnet** [comment]
@rakekniven I have made changes to those files where there is a translation in Transifex and in the Talk documentation.

File: docs/call-experience.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Thumbnail | 320 | - | 240 | - | 1 | -
 
 ## Judging the connection quality
 
-Similar since version 9.0.2 Nextcloud Talk is having an eye on the lost packages and the "round trip time" of the stream data. When the connection is too bad or no data could be transmitted at all, the participant will be informed to try to disable their own video and screenshare.
+Similar since version 9.0.2 Nextcloud Talk is having an eye on the lost packages and the "round trip time" of the stream data. When the connection is too bad or no data could be transmitted at all, the participant will be informed to try to disable their own video and sharing screen.
```
Comment (rakekniven) [inline docs/call-experience.md:44]: "sharing screen" -> "screen sharing"

File: docs/constants.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ title: Constants
 * `8` Can ignore lobby
 * `16` Can publish audio stream
 * `32` Can publish video stream
-* `64` Can publish screenshare stream
+* `64` Can publish sharing screen stream
```
Comment (rakekniven) [inline docs/constants.md:72]: "sharing screen" -> "screen sharing"

File: src/components/CallView/shared/LocalMediaControls.vue [hunk 4]
```diff
@@ -559,7 +559,7 @@ export default {
 
 			const tooltip = {}
 			if (!this.model.attributes.audioEnabled && this.model.attributes.videoEnabled && virtualBackgroundEnabled && this.model.attributes.localScreen) {
-				tooltip.content = t('spreed', 'Your internet connection or computer are busy and other participants might be unable to see your screen. To improve the situation try to disable the background blur or your video while doing a screenshare.')
+				tooltip.content = t('spreed', 'Your internet connection or computer are busy and other participants might be unable to see your screen. To improve the situation try to disable the background blur or your video while doing a sharing screen.')
```
Comment (rakekniven) [inline src/components/CallView/shared/LocalMediaControls.vue:562]: "sharing screen" -> "screen sharing"

**Review by rakekniven — CHANGES_REQUESTED** [review]
Sorry für picking here but the grammar needs one more change.

File: docs/constants.md (hunk 3 above)
Comment (Valdnet) [reply docs/constants.md]: Changed 😉.

File: src/components/CallView/shared/LocalMediaControls.vue (hunk 4 above)
Comment (Valdnet) [reply src/components/CallView/shared/LocalMediaControls.vue]: Changed 😉.

File: docs/call-experience.md (hunk 2 above)
Comment (Valdnet) [reply docs/call-experience.md]: Changed 😉.

**Comment by Valdnet** [comment]
@rakekniven Changed. Please approval 😉.

File: src/components/ConversationSettings/ConversationPermissionsSettings.vue [hunk 5]
```diff
@@ -52,7 +52,7 @@
 			</CheckboxRadioSwitch>
 			<span v-show="loading && radioValue === 'restricted'" class="icon-loading-small" />
 		</div>
-		<p>{{ t('spreed', 'Participants can only join calls, but not enable audio, video or screenshare until a moderator manually grants their permissions manually.') }}</p>
+		<p>{{ t('spreed', 'Participants can join calls, but cannot enable audio and video or share screen until a moderator manually grants their permissions manually.') }}</p>
```
Comment (nickvergessen) [inline src/components/ConversationSettings/ConversationPermissionsSettings.vue:55]:
```suggestion
		<p>{{ t('spreed', 'Participants can join calls, but cannot enable audio nor video nor share screen until a moderator manually grants them permissions.') }}</p>
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by rakekniven — APPROVED** [review]

**Comment by Valdnet** [comment]
@nickvergessen Please approve, squash and merge. Thanks 😉.

File: src/components/CallView/shared/LocalMediaControls.vue [hunk 6]
```diff
@@ -559,7 +559,7 @@ export default {
 
 			const tooltip = {}
 			if (!this.model.attributes.audioEnabled && this.model.attributes.videoEnabled && virtualBackgroundEnabled && this.model.attributes.localScreen) {
-				tooltip.content = t('spreed', 'Your internet connection or computer are busy and other participants might be unable to see your screen. To improve the situation try to disable the background blur or your video while doing a screenshare.')
+				tooltip.content = t('spreed', 'Your internet connection or computer are busy and other participants might be unable to see your screen. To improve the situation try to disable the background blur or your video while doing a screen sharing.')
```
Comment (nickvergessen) [inline src/components/CallView/shared/LocalMediaControls.vue:562]:
```suggestion
				tooltip.content = t('spreed', 'Your internet connection or computer are busy and other participants might be unable to see your screen. To improve the situation try to disable the background blur or your video while doing a screen share.')
```

Not a native speaker, but this sounds better to me. You do a "Screen share" not a "Screen sharing"

Comment (Valdnet) [reply src/components/CallView/shared/LocalMediaControls.vue]: Done 😉.

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**